source/scripting/World/World.cs

```diff
@@ -605,11 +605,15 @@ public static Blip[] GetAllBlips(params BlipSprite[] blipTypes)
 		/// Gets an <c>array</c> of all the <see cref="Checkpoint"/>s.
 		/// </summary>
 		public static Checkpoint[] GetAllCheckpoints()
-		{					   
+		{
 			return Array.ConvertAll<int, Checkpoint>(MemoryAccess.GetCheckpointHandles(), element => new Checkpoint(element));
 		}
 
-		static int[] ModelListToHashList(Model[] models)
+		/// <summary>
+		/// Gets the hash of all <see cref="Model"/>s in a list
+		/// </summary>
+		/// <param name="models">A list of <see cref="Model"/>s</param>
+		public static int[] ConvertModelListToHashList(Model[] models)
```

Comment:
I'm not convinced this should be `public`.

Reply:
So we should expose the methods without giving the end-user a proper way to convert if they want to? Just seemed logical to expose a useful utility method. But it might be out of scope. Thoughts?

Reply:
Even in local scripts, you can achieve this with `Array.ConvertAll` or LINQ.
An example for LINQ
```
var hashes = models.Select(x => x.Hash).ToArray();
```
And you don't need to change the method name.

Reply:
Hmm, this PR's main purpose seems to be to overload some methods for `int[]` params, but you don't need to.
You formatted a bit, but I didn't noticed at first. If you made different commits for this, other people could notice easily.
That's one of the reasons why you shouldn't include (too many) different kinds of changes in one commit.

Reply:
I'm fine with the additional overloads, since it's faster to call them if you already have a hash list with integers. Just don't want to polute `World` with helper methods like the one mentioned.

Reply:
Oh, didn't noticed the additional overloads have the merit.

Reply:
Okay, I'll remove the utility function again :)